lblod/mandataris-service
# new version of handling besluit information

## Description

Besluiten are now handled as described on the gitbook: https://app.gitbook.com/o/-MP9Yduzf5xu7wIebqPG/s/UYHyp4MV7ApSjjQn2DUx/algemene-informatie/besluit-harvesting

## How to test

Use the mock route to receive besluit instances in the consumer graph. Verify that indeed they are processed as per the gitbook.

After each mock route check, run the clear decisions endpoint: localhost:8082/mock/clear-decisions

The mock routes are configured for use in provincie vlaams brabant as an example bestuurseenheid. Because of a delay of delta message processing you need to wait 10s before you see the notifications.

mock routes exist for:
- http://localhost:8082/mock/add-decision: default case, nothing special. Changes status to bekrachtigd and adds an info notification
- http://localhost:8082/mock/add-decision?new=true: unknown mandataris, created on our end with notification
- http://localhost:8082/mock/add-decision?fraction=true: unknown fraction, error message
- http://localhost:8082/mock/add-decision?unknownMandaat=true: unknown mandaat, error message
- http://localhost:8082/mock/add-decision?incompletePerson=true: incomplete person, error message
- http://localhost:8082/mock/add-decision?changeFractionName=true: different fraction name
- http://localhost:8082/mock/add-decision?newBeleidsdomein=true: new beleidsdomein, info message

## Links to other PR's

- https://github.com/lblod/app-lokaal-mandatenbeheer/pull/212
